docs/product/performance/web-vitals/index.mdx

```diff
@@ -38,7 +42,7 @@ The **Web Vitals** page only displays data from your application's initial page
 
 ## Page Overview
 
-From the table in **Web Vitals** or a Web Vital Summary panel, click on a page to open its **Page Overview**. Here, you can see summary of that single page's Web Vitals. In **Page Overview**, you can further drill down to a specific page load sample [Event](/product/sentry-basics/concepts/tracing/event-detail/), [Replay](/product/session-replay/), or [Profile](/product/profiling/).
+From the table in **Web Vitals** or a Web Vital Summary panel, click on a page to open its **Page Overview**. Here, you can see a summary of that single page's Web Vitals. In **Page Overview**, you can further drill down to a specific page load sample [Event](/product/sentry-basics/concepts/tracing/event-detail/), [Interaction](), [Replay](/product/session-replay/), or [Profile](/product/profiling/).
```

Comment:
@edwardgou-sentry Did you have a link you wanted to add for interaction?

Reply:
good catch, fixed!